browser/src/map/handler/Map.WOPI.js

```diff
@@ -335,6 +335,11 @@ L.Map.WOPI = L.Handler.extend({
 				this._map._disableDefaultAction[msg.Values.action] = msg.Values.disable;
 			}
 		}
+		else if (msg.MessageId === 'Error_Messages') {
+			if (msg.Values.websocketconnectionfailed) {
+				errorMessages.websocketconnectionfailed = _(msg.Values.websocketconnectionfailed);
```

Comment:
if any custom message will come here, we will not be able to translate it - because it will not be found  in our translations

Reply:
@eszkadev right, I mean if the string comes from outside, sure we cannot guarantee it's translated. But probably that should already come translated from the integrator. (Don't know if it's possible but I would expect integrators to set the value of that message differently according to user's UI )